definitions/pipelines/detect_variants.cwl

```diff
@@ -118,6 +118,10 @@ inputs:
     custom_clinvar_vcf:
         type: File?
         secondaryFiles: [.tbi]
+    tumor_sample_name_for_mutect2:
+        type: string
+    normal_sample_name_for_mutect2:
+        type: string
```

Comment:
Let's just call these "tumor_sample_name" and "normal_sample_name" and get them from the "tumor_name/normal_name" fields in upstream places like somatic_exome.cwl.  Alternately, I guess we could infer them from the header of the bam, since they're required to match the bam?

Reply:
The more I think about this, the more I think that you should grab those sample names from the headers of the bams. No matter what gets input into your sample name fields here, mutetech is going to base the headers of the VCF columns off of the sample names present in the header of the bam. If you want to be guaranteed that they will match, then you should pull them from the same place.